ksql-rest-app/src/test/java/io/confluent/ksql/rest/client/KsqlRestClientTest.java

```diff
@@ -76,11 +77,71 @@ public void testKsqlResource() {
 
 
   @Test
-  public void testStreamQuery() {
+  public void testStreamQuery() throws InterruptedException {
```

Comment:
What is this testing? The name doesn't tell me much and The test is hard to follow.

Reply:
It verifies that we can receive a row sent by the server. I'll update the name and add some comments